doc/sphinx/changes.rst

```diff
@@ -14,6 +14,91 @@ Version 8.12
+- A correct and efficient `coercion coherence`__ checking algorithm, avoiding
+  spurious or duplicate warnings.
+- An improved :g:`Search` `command`__ which accepts complex queries. Note that
+  this takes precedence over the now deprecated `ssreflect search`__.
+- Many additions and improvements of the `standard library`__.
+- Overhaul of the `reference manual`__ giving a more coherent view of the system,
+  along with consistent grammar notations that are up-to-date with Coq's grammar.
+
+__ 812Implicits_
+__ 812Uniform_
+__ 812ImplicitInductive_
+__ 812Notations_
+__ 812Coercions_
+__ 812Search_
+__ 812SSRSearch_
+__ 812Stdlib_
+__ 812Refman_
+
+Additionally, the :tacn:`omega` tactic is deprecated in this version of Coq,
+and we recommend users to switch to :tacn:`lia` in new proof scripts (see
+also the warning message in the :ref:`corresponding chapter
+<omega_chapter>`).
+
+See the `Changes in 8.12+beta1`_ section and following sections for the
+detailed list of changes, including potentially breaking changes marked
+with **Changed**.
+
+Coq's documentation is available at https://coq.github.io/doc/v8.12/refman (reference
+manual), and https://coq.github.io/doc/v8.12/stdlib (documentation of
+the standard library). Developer documentation of the ML API is available
+at https://coq.github.io/doc/v8.12/api.
+
+Maxime Dénès, Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop and Théo Zimmermann worked on maintaining and improving the
+continuous integration system and package building infrastructure.
+
+The OPAM repositoryy for |Coq| packages has been maintained by
+Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
+contributions from many users. A list of packages is available at
+https://coq.inria.fr/opam/www/.
+
+The 59 contributors to this version are Abhishek Anand, Yves Bertot, Frédéric
+Besson, Lasse Blaauwbroek, Simon Boulier, Quentin Carbonneaux, Tej Chajed,
+Arthur Charguéraud, Cyril Cohen, Pierre Courtieu, Matthew Dempsky, Maxime Dénès,
+Andres Erbsen, Erika (@rrika), Nikita Eshkeev, Jim Fehrle, @formalize, Emilio
+Jesús Gallego Arias, Paolo G. Giarrusso, Gaëtan Gilbert, Jason Gross, Samuel
+Gruetter, Attila Gáspár, Hugo Herbelin, Jan-Oliver Kaiser, Robbert Krebbers,
+Vincent Laporte, Olivier Laurent, Xavier Leroy, Thomas Letan, Yishuai Li, Xia
+Li-yao, Kenji Maillard, Erik Martin-Dorel, Guillaume Melquiond, Ike Mulder,
+Guillaume Munch-Maccagnoni, Antonio Nikishaev, Karl Palmskog, Clément
+Pit-Claudel, Pierre-Marie Pédrot, Ramkumar Ramachandra, Lars Rasmusson, Daniel
+de Rauglaudre, Talia Ringer, Pierre Roux, Kazuhiko Sakaguchi, Vincent Semeria,
+@scinart, Kartik Singhal, Michael Soegtrop, Matthieu Sozeau, Enrico Tassi,
+Laurent Théry, Ralf Treinen, Anton Trunov, Bernhard M. Wiedemann, Nickolai
+Zeldovich and Théo Zimmermann.
+
+Many power users helped to improve the design of this new version via
+the issue and pull request system, the |Coq| development mailing list,
```

Comment:
```suggestion
the GitHub issue and pull request system, the |Coq| development mailing list,
```

Perhaps obvious but might as well be specific as you are for the other tools

Maybe also give the email address for the dev mailing list?

Reply:
Right, done.